test/integration/default/verify/controls/aws_vpc.rb

```diff
@@ -0,0 +1,17 @@
+fixtures = {}
+[
+  'ec2_security_group_default_vpc_id',
+].each do |fixture_name|
+  fixtures[fixture_name] = attribute(
+    fixture_name,
+    default: "default.#{fixture_name}",
+    description: 'See ../build/ec2.tf',
+  )
+end
+
+control "aws_vpc default vpc" do
```

Comment:
Please split into sections: searching, then exercising properties and matchers.

Reply:
I think we need to make a non-default VPC so we can test the is_default matcher.

Reply:
We need a search miss test.